RELEASENOTES/v2.0.0.alpha-3.MD

```diff
@@ -0,0 +1,59 @@
+    - Setup Trust stores with the new directory-based structure
+    - Configure Trust Policy as a JSON document. Support for registry scope and signature verfication levels to customize the behaviour during verification.
+    - Store signatures in registries compliant to the ORAS Reference spec [RC-1](https://github.com/oras-project/oras-go/releases/tag/v2.0.0-rc.2)
+
+*We do not recommend this release for use in production environment. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). We suggest using it for testing and providing feedback to the Notary v2 working group only*
+
+## List of feature/capabilities added with this release
+For detailed notes on each of the individual repositories (NotaryProject, Notation (for reference CLI implementation) and Notation libraries) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notary v2 specifications
+Updated Trust Policy, Trust Store,  Signature Verification and Signing workflows along with extensibility mechanisms via Notation plugins.
+- A final JWS based [signature envelope format.](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md)
+- Updated [Signature specfication](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md) with extensibility provisions for additional signature envelope format.
+- Updated [signing and verification workflows](https://github.com/notaryproject/notaryproject/blob/main/signing-and-verification-workflow.md)
+- Introducing a new [Signing scheme](https://github.com/notaryproject/notaryproject/blob/main/signing-scheme.md) to integrate Notation Client with Signing Services
+- Updated  [trust store and trust policy](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md) specification
+
+### Notation CLI Client
+Notation is a standalone CLI client based on Notary v2 specs. It supports signing, verifying, and storing signatures in [oras-artifacts-spec](https://github.com/oras-project/artifacts-spec/) compliant registries. Starting with this release of Notation, the trust store will be based on a directory structure and trust policy as a JSON document. Users can simply insert  the public certificates inside pre-configured directories for configuring their trust store, and modify the trust policy document to granularly control the scope and behavior of the signature verification policies. The use of the existing CLI commands (introduced in the alpha-1 version of Notation client when the keys/certificates for signing and verifying were specified in a file) has been subsituted by use of a directory based trust store.  Refer https://github.com/notaryproject/notation/blob/main/specs/notation-config.md.
+
+In this release of Notation command, the following CLI commands are supported. Refer (TBD - Getting started here)
+- notation sign  
```

Comment:
We should add support for `notation cert generate-test` https://github.com/notaryproject/notation/issues/252

Reply:
added